anton-melyakov/homework1/CV anton/main.css

```diff
@@ -0,0 +1,185 @@
+body{
+	
+	color: black;
```

Comment:
You probably want your text color to be white, because the background is black : )